website/source/api/acl-legacy.html.md

```diff
@@ -12,7 +12,8 @@ the new ACL [Token](/docs/api/acl-token.html) and [Policy](/docs/api/acl-policy.
 
 # ACL HTTP API
 
-These `/acl` endpoints create, update, destroy, and query ACL tokens in Consul. For more information about ACLs, please see the [ACL Guide](/docs/guides/acl.html).
+These `/acl` endpoints create, update, destroy, and query ACL tokens in Consul. For more information about ACLs, please see the 
+[ACL Guide](https://learn.hashicorp.com/consul/security-networking/production-acls).
```

Comment:
I wonder if there is a `middleman` way to have a list of general named links stored somewhere that can be referred to from other docs to avoid having to inline them all over the place like this.

Reply:
pinging @jescalan for any ideas ^

Reply:
The downside of storing them separately is the possible confusing of figuring out where a link goes. It could add another step when editing docs too, editors would need to go to another file to track down links.